### ISSUE TITLE
Converter skipped mysql_set_charset function

### ISSUE THREAD

**Issue by semoriil** [philip/MySQLConverterTool#13]
Was
```
  $link = mysql_connect(HostName, UserName, Password);
  if (!$link)
  {
    exit("Error connecting to DB! " . mysql_error());
  }
  mysql_select_db(DBName, $link);
  if (!mysql_set_charset('utf8', $link))
    echo 'Setting charset error: ' . mysql_error();
```
converted to
```
  $link = ($GLOBALS["___mysqli_ston"] = mysqli_connect(HostName,  UserName,  Password));
  if (!$link)
  {
    exit("Error connecting to DB! " . ((is_object($GLOBALS["___mysqli_ston"])) ? mysqli_error($GLOBALS["___mysqli_ston"]) : (($___mysqli_res = mysqli_connect_error()) ? $___mysqli_res : false)));
  }
  ((bool)mysqli_query( $link, "USE " . constant('DBName')));
  if (!mysql_set_charset('utf8', $link))
    echo 'Setting charset error: ' . ((is_object($GLOBALS["___mysqli_ston"])) ? mysqli_error($GLOBALS["___mysqli_ston"]) : (($___mysqli_res = mysqli_connect_error()) ? $___mysqli_res : false));
```
Shoul be replaced with something like
```
...
if ((!is_object($GLOBALS["___mysqli_ston"])) || !mysqli_set_charset ( $GLOBALS["___mysqli_ston"], 'utf8' ))
...
```

**Comment by tanloverluke** [comment]
when you pass these through the converted are you wrapping them in <?php ... ?> ?

**Comment by tanloverluke** [comment]
I must admit I gave this conversion tool a go last night and it worked very well. However in the end I decided to make the changes myself because I didn't like the globals being set. Took a lot longer but the end result is a lot cleaner.

**Comment by SergeAx** [comment]
I confirm this issue.

**Comment by philip** [comment]
This was fixed via pull request #17 -- thank you for filing this bug report!
